scripts/module.js

```diff
@@ -32,6 +32,7 @@ class S3Utils {
         
         if(game.settings.get(this.ID, this.SETTINGS.CUSTOM_STYLE)){
             game.data.files.s3.endpoint.hostname = game.settings.get(this.ID, this.SETTINGS.CUSTOM_PREFIX);
+            game.data.files.s3.endpoint.port = game.settings.get(this.ID, this.SETTINGS.CUSTOM_PREFIX); // No idea what this does, truth be told.
```

Comment:
```suggestion
```

This line should be removed. If that setting is ever set to anything, it'd blow up the port.